### PR TITLE
Add normalizer for struct fields.

### PR DESCRIPTION
**Pull request by sepal**
This PR adds support to export "struct" fields, used for example for date intervals. Any other ideas where this is used?

### CODE REVIEW

File: include/struct_normalizer.inc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * Provides the date normalizer
+ */
+
+use \Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * A data formatter for date columns.
```
Comment (klausi) [inline include/struct_normalizer.inc]: why is it called "struct" when it is for dates? Probably the comments here need updating.

**Comment by klausi** [comment]
I think entity structure wrappers are used for all fields that have multiple columns. Like long text fields that have text + format.

**Comment by sepal** [comment]
> I think entity structure wrappers are used for all fields that have multiple columns. Like long text fields that have text + format.

Unfortunally no, formatted text uses a custom type called `text_formatted`. It does however use the `EntityStructureWrapper` class, which means that currently such fields aren't exported, since the `GDPRExportGenericDataNormalizer` checks for a value wrapper and not a structure wrapper. I'll make a seperate PR to fix that issue.

**Review by klausi — APPROVED** [review]
Looks good!

**Pull request merged**